libnl-route-3.sym

```diff
@@ -1331,4 +1331,10 @@ global:
 	rtnl_route_get_nhid;
 	rtnl_route_nh_identical;
 	rtnl_route_set_nhid;
+	rtnl_link_get_perm_addr;
```

Comment:
The unit tests fail, see the github actions.
I think it's due to this line (I suspect you can drop this line).

Reply:
indeed, some leftover while testing. removed. CI is passing now